examples/label_studio_annotation/README.md

```diff
@@ -0,0 +1,318 @@
+# 🏷 Continuous Annotation with Label Studio
+
+Data annotation / labeling is a core part of MLOps that is frequently left out
+of the conversation. ZenML offers a way to build continuous annotation (combining
+training and annotation into a loop) with Label Studio. This uses a combination
+of user-defined steps as well as some built-in steps that ZenML provides.
+
+## Basic guide to running this example
+
+This example requires a few more steps than usual, given that it requires a
+cloud artifact store. Instructions for Azure, AWS S3 and GCP can be found in the
+following guide.
+
+## 🗺 Overview
+
+NOTE: This example currently only runs with a cloud artifact store. [This guide
+by Label Studio](https://labelstud.io/guide/storage.html) contains more details
+if you have trouble understanding the instructions below.
+
+For a full video walkthrough on how to run this email, please check out [our
+community hour demo](https://www.youtube.com/watch?v=bLFGnoApWeU) in which we
+show how to run the various pipelines, in what order, all using an AWS stack.
+
+# 🖥 Run the example
+
+## 👣 Step-by-Step
+
+### 🥞 Set up your stack for Microsoft Azure
+
+In order to run this example, you need to install and initialize ZenML and Label
+Studio.
+
+```shell
+zenml integration install label_studio pytorch azure
+
+# pull example
+zenml example pull label_studio_annotation
+cd zenml_examples/label_studio_annotation
+
+# Initialize ZenML repo
+zenml init
+```
```

Comment:
This is essentially the same for all three cloud providers, only `zenml integration install azure` is provider-specific here. I would recommend to move the rest into a separate section "ZenML Setup" or similar to remove code duplication and make the README a bit shorter

Reply:
Yep I moved that now.